### PR TITLE
Update UI to CRB-inspired theme

### PR DESCRIPTION
**Pull request by Dorako**
CleanShot 2022-12-03 at 15 13 29@2x

Looks like this.

Not sure whether to hide it behind a toggle, or how I'd go about doing that in a reasonable way.
Also not sure whether to include the lines relating to module compatibility, or just put in the module.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,5 @@ packs/temp-data
 static/packs/*
 types/foundry/node_modules/
 types/foundry/package-lock.json
+.DS_Store
```
Comment (Dorako) [inline .gitignore:17]: macOS spawns these files everywhere, so I think they should be excluded

File: src/styles/_colors.scss [hunk 2]
```diff
@@ -16,6 +18,7 @@ $tertiary-color: #E9D7A1 !default; /* compliments the primary color, usually use
 
 $alt-color: #786452 !default; /* for all other uses */
 $alt-dark: #443730 !default;
+$gold-brown: #956d58 !default;
```
Comment (Dorako) [inline src/styles/_colors.scss:21]: This color is used by mdizo several times, and I'm also using it, so I figured it should probably be a variable

File: src/styles/_colors.scss [hunk 3]
```diff
@@ -105,4 +113,22 @@ $adjusted-lower: #cc3311 !default;
   /* Sidebar */
   --sidebar-label: var(--tertiary-dark);
   --sidebar-title: var(--tertiary);
+
+  /* Box shadows */
```
Comment (Dorako) [inline src/styles/_colors.scss:117]: These don't *really* fit here, but I couldn't think of a better place

File: src/styles/_globals.scss [hunk 4]
```diff
@@ -265,22 +260,17 @@ span[data-pf2-effect-area] {
 
 i[data-pf2-repost] {
```
Comment (Dorako) [inline src/styles/_globals.scss:261]: The new repost can be seen in the image in the description

File: src/styles/mixins/_frames.scss [hunk 5]
```diff
@@ -1,9 +1,35 @@
...
+    /* prettier-ignore */
+    border: none;
+    margin: 2px;
+    box-shadow:
+        0 0 0 1px var(--tertiary),
+        0 0 0 2px var(--gold-brown),
+        0 0 20px rgba(black, 0.8);
+}
+
 @mixin gold-border {
```
Comment (Dorako) [inline src/styles/mixins/_frames.scss:11]: This style was only used one place, the trim around the currencies. The new style is 1px thinner and is used in a lot of locations.

File: .gitignore (hunk 1 above)
Comment (simonward) [reply .gitignore:17]: If they're excluded, they should be ignored when building.
https://github.com/foundryvtt/pf2e/pull/4816

File: src/styles/_globals.scss (hunk 4 above)
Comment (Dorako) [reply src/styles/_globals.scss]: Removed this stuff from this PR and put it in https://github.com/foundryvtt/pf2e/pull/5132

**Comment by Dorako** [comment]
I'm closing this and replacing it with multiple smaller-scoped PR's, starting with https://github.com/foundryvtt/pf2e/pull/5762

**Pull request closed without merging**